src/app/amsterdam/contact/gegevens/page.tsx

```diff
@@ -32,47 +33,45 @@ function Question() {
       <Grid.Cell span={{ narrow: 4, medium: 6, wide: 8 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
         <Breadcrumb>
           <NextLink legacyBehavior passHref href="/amsterdam">
-            <Breadcrumb.Item href="/amsterdam">Home</Breadcrumb.Item>
+            <Breadcrumb.Link href="/amsterdam">Home</Breadcrumb.Link>
           </NextLink>
         </Breadcrumb>
         <form className="ams-gap--md" onSubmit={handleSubmit}>
           <Heading>Contact</Heading>
-          <Fieldset legend="Wat zijn uw contactgegevens?">
-            <Column gap="small">
-              <div>
-                <Paragraph>Wij hebben uw gegevens nodig om contact met u te kunnen opnemen.</Paragraph>
-              </div>
-              <div>
+          <FieldSet legend="Wat zijn uw contactgegevens?">
+            <Column gap="small" style={{ clear: 'both' }}>
+              <Paragraph>Wij hebben uw gegevens nodig om contact met u te kunnen opnemen.</Paragraph>
+              <Field>
```

Comment:
It's cool that a Field in a FieldSet works, but it isn't really an intended use. Do we want to make it one? We don't have any documentation on this yet.

Reply:
I would think so, yes. `FieldSet` is the usual way to group a set of radio buttons and give the group a title, but it can likewise group a set of related text inputs. These fields will still need their label, description and error message, which `Field` organizes.

Reply:
I agree, I've added [a ticket to fix this in the FieldSet stories](https://gemeente-amsterdam.atlassian.net/browse/DES-801)